### DISCUSSION TITLE
August 2019 report: Fion, Plakar and OpenSMTPD · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#104] [Articles]

**Comment by whataboutpereira** [comment]
Very much enjoy reading about your activities and thoughts. :)

**Comment by poolpOrg** [comment]
Thanks :-)

**Comment by ngortheone** [comment]
I listened to FLOSS Weekly podcast about OpenSMTPD. Really liked it, apart from one thing - the show's host was an ignorant idiot, who couldn't spell your name right even once, and was especially pleased by that.

@RandalSchwartz, if you read this - I think you need to apologize for the disrespect that you demonstrated during the show. And next time try to have some more respect to your guests (by learning their name, for starters)

**Comment by poolpOrg** [comment]
I'm glad you liked the show :-)

I don't think there was disrespect from him and didn't feel it this way personally, we had a chat before the show and he was really nice to me. People having trouble with my name is not uncommon and I don't have habit of correcting because even in my own language people mispronounce so I don't expect foreign people to pronounce correctly.

I've been in the OpenBSD project for over a decade and some people genuinely think they pronounce it correctly despite having mispronounced it for a decade ;-)
